src/command.c

```diff
@@ -464,54 +465,62 @@ static int dv_convert_conv_v0(struct device *dev, struct dmp_cmb *cmb,
 		conv->run[i].rectifi_en = cmd->run[i].rectifi_en;
 		conv->run[i].lrn = cmd->run[i].lrn;
 
-		if ((conv->run[i].lrn & 1) && ((conv->run[i].conv_enable) || (conv->run[i].pool_enable) || (conv->header.topo != 1))) {
+		if ((conv->run[i].lrn & 1) && ((conv->run[i].conv_enable)
+			|| (conv->run[i].pool_enable)
+			|| (conv->header.topo != 1))) {
 			kfree(cmd);
 			pr_warn(DRM_DEV_NAME ": LRN must be a standalone layer\n");
 			return -EINVAL;
 		}
 
-		if ((cmd->z > 1) || (cmd->run[i].pz > 1) || (cmd->run[i].conv_dilation))  // TODO: add more checks: no maxpool_with_argmax, no unpool_with_argmax.
+		if ((cmd->z > 1)
+			|| (cmd->run[i].pz > 1)
+			|| (cmd->run[i].conv_dilation))  // TODO: add more checks: no maxpool_with_argmax, no unpool_with_argmax.
 			valid_multi_run = 0;
 	}
 	if (output_buf_size < total_output_size ||
-	    (eltwise_base_addr != 0xDEADBEEF &&
-	     eltwise_buf_size < total_output_size)) {
+			(eltwise_base_addr != 0xDEADBEEF &&
+			 eltwise_buf_size < total_output_size)) {
 		kfree(cmd);
 		pr_warn(DRM_DEV_NAME ": got eltwise buffer size %u while expected %u\n",
-			eltwise_buf_size, total_output_size);
+				eltwise_buf_size, total_output_size);
 		return -EINVAL;
 	}
 	if (cmd->topo != 1) {
 		if (!valid_multi_run) {
 			pr_warn(DRM_DEV_NAME ": Configuration %dx%dx%d z=%d cannot be executed with multiple runs\n",
-				(int)conv->input.w, (int)conv->input.h, (int)conv->input.c, (int)conv->input.z);
+					(int)conv->input.w, (int)conv->input.h,
+					(int)conv->input.c, (int)conv->input.z);
 			kfree(cmd);
 			return -EINVAL;
 		}
-		ubuf_size = ubuf_get_single_tile_usage((struct dmp_dv_kcmdraw_conv_v0*)cmd, UNIFIED_BUFFER_SIZE);
+		ubuf_size = ubuf_get_single_tile_usage(
+				(struct dmp_dv_kcmdraw_conv_v0*)cmd,
+				UNIFIED_BUFFER_SIZE);
 		if (ubuf_size > UNIFIED_BUFFER_SIZE) {
 			kfree(cmd);
 			pr_warn(DRM_DEV_NAME ": Configuration %dx%dx%d requires unified buffer of size %d\n",
-				(int)conv->input.w, (int)conv->input.h, (int)conv->input.c, (int)ubuf_size);
+					(int)conv->input.w, (int)conv->input.h,
+					(int)conv->input.c, (int)ubuf_size);
 			return -EINVAL;
 		}
 	}
 	/*pr_info(DRM_DEV_NAME ">>>\n");
-	for (i = 0; i < (cmd_size >> 2); ++i)
-		pr_info(DRM_DEV_NAME ": %u: %08X\n", i << 2, cmd_buf[i]);
-	pr_info(DRM_DEV_NAME "<<<\n");*/
+	  for (i = 0; i < (cmd_size >> 2); ++i)
+	  pr_info(DRM_DEV_NAME ": %u: %08X\n", i << 2, cmd_buf[i]);
+	  pr_info(DRM_DEV_NAME "<<<\n");*/
 	/*if (conv->run[0].lrn)
-		pr_info(DRM_DEV_NAME ": LRN tiles: %dx%dx%d: %d\n",
-			(int)conv->input.w, (int)conv->input.h, (int)conv->input.c,
-			(int)conv->input.tiles);*/
+	  pr_info(DRM_DEV_NAME ": LRN tiles: %dx%dx%d: %d\n",
+	  (int)conv->input.w, (int)conv->input.h, (int)conv->input.c,
+	  (int)conv->input.tiles);*/
 	cmb_node->size += cmd_size;
 
 	kfree(cmd);
 	return 0;
 }
 
 int dv_convert_conv_command(struct device *dev, struct dmp_cmb *cmb,
-			    struct dmp_dv_kcmd *cmd_info)
+			struct dmp_dv_kcmd *cmd_info)
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket